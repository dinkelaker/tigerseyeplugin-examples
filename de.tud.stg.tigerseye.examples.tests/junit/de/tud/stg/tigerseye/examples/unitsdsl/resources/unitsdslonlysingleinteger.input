	println (  18 kg)
	println ( 18 kg * 2 )
	println ( 1800000 kg / 3 ) 
	println ( 1 kg * 2 + 3 kg / 4 )
	println ( 3 cm + 12 m * 3 - 1 km )
	println ( 18 kg * 2 )
	println ( 1800000 kg / 3 )
	println ( 1 kg * 2 + 3 kg / 4 )
	println ( 1 h + 33 s - 3 h )	