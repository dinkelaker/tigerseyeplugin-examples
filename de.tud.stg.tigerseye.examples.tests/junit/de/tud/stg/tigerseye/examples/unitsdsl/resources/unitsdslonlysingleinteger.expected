println (  kilogram(
18))
	println ( kilogram(
18) * 2 )
	println ( kilogram(
1800000) / 3 ) 
	println ( kilogram(
1) * 2 + kilogram(
3) / 4 )
	println ( centimeter(
3) + meter(
12) * 3 - kilometer(
1) )
	println ( kilogram(
18) * 2 )
	println ( kilogram(
1800000) / 3 )
	println ( kilogram(
1) * 2 + kilogram(
3) / 4 )
	println ( hours(
1) + seconds(
33) - hours(
3) )	